client/src/components/AuthNavbar/useStyles.tsx

```diff
@@ -60,18 +46,63 @@ const useStyles = makeStyles((theme) => ({
   notificationsPopper: {
     pointerEvents: 'none',
     borderTop: '.25rem solid black',
-    width: '300px',
+    width: '450px',
     backgroundColor: 'white',
     border: '1px solid lightgrey',
     borderRadius: '5px',
+    zIndex: 9999,
   },
   profilePopper: {
     backgroundColor: 'white',
     border: '1px solid lightgrey',
     borderRadius: '5px',
+    zIndex: 9999,
+  },
+  badgeIcon: {
+    '& .MuiBadge-anchorOriginTopRightCircle': {
+      left: 0,
+    },
+  },
```

Comment:
It might be more better to just move this style directly on the element as opposed to grabbing by the class name which is not guaranteed to be the same in the future.

Reply:
Will do that